NMS-14237: Document log level impact (#4678)
OpenNMS/opennms: docs/modules/deployment/pages/logging.adoc

> === Debugging

Reviewer: This page seems like a duplicate of data in operations/admin/logging.  Not sure why it's part of the Deployment section.

If this page does stay, suggestions below align the doc with the style guide for structuring files.

```suggestion

[[ga-deployment-debugging]]
=== Debugging
```

Reviewer 2: I agree with Mark. If we keep it, I suggest keeping the first sentence, "When debugging, it may be helpful to lower the minimum severity at which a component will log from the default level of WARN. See xref:operation:admin/logging/introduction.adoc [Logging] for more information and instructions on how to change logging levels.

Reviewer 2: @fooker , this PR has been in my queue for a while. Could we add the cross reference to the logging section, then merge?

When debugging, it may be helpful to lower the minimum severity at which a component will log from the default level of `WARN`.
To change this setting, edit `$\{OPENNMS_HOME}/etc/log4j2.xml` and locate the following lines:

[source,xml]
----
<KeyValuePair key="..." value="WARN" />
----

> The log level can be altered to the following values: `DEBUG`, `INFO`, `WARN`, `ERROR`.

Reviewer 2: ```suggestion
You can alter the log level to the following values: `DEBUG`, `INFO`, `WARN`, `ERROR`.
```

> Changes to `log4j2.xml` will take effect within 60 seconds with no extra action needed.

Reviewer 2: ```suggestion
Changes to `log4j2.xml` take effect within 60 seconds with no extra action needed.
```

> On busy systems, setting the log level to `DEBUG` may create so much noise as to be impractical or even have performance impacts.

Reviewer: ```suggestion
On busy systems, setting the log level to `DEBUG` may create so much noise as to be impractical or even have performance impacts.

```
